### PR TITLE
[a11y] NuGet Package Management

### PR DESCRIPTION
**Pull request by sevoku**
This fixes some accessibility issues around package management. This is not ideal, and to make it really right we need to make this native. However the fixes here make it good enough to be MAS compliant.

Fixes VSTS #750371
Fixes VSTS #750376

### CODE REVIEW

File: main/src/addins/MonoDevelop.PackageManagement/MonoDevelop.PackageManagement.Gui/ManagePackagesDialog.cs [hunk 1]
```diff
@@ -105,9 +106,13 @@ public ManagePackagesDialog (ManagePackagesViewModel viewModel, string initialSe
 			imageLoader.Loaded += ImageLoaded;
 
 			browseLabel.ButtonPressed += BrowseLabelButtonPressed;
+			browseLabel.KeyPressed += BrowseLabelKeyPressed;
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.PackageManagement/MonoDevelop.PackageManagement.Gui/ManagePackagesDialog.cs]: Unsubscribe on dispose

**Comment by sevoku** [comment]
@monojenkins backport release-8.4

**Comment by sgmunn** [comment]
tried it out, VO appears to work as expected.

**Comment by sevoku** [comment]
@mrward thanks! Yeah the spacing is not ideal, but TBH it was not perfect before either. The contrast problem affects all Gtk tables unfortunately, we need to improve colors there. I think the next step will be native, I tried to get it working, but the main problem for me was the toolbar (which is a custom backend thing), so I ran out of time for the fix.

**Pull request merged**